tests/cpputest/jakstat_adjoint/tests1.cpp

```diff
@@ -6,6 +6,7 @@
 #include <cstring>
 #include "wrapfunctions.h"
 
+#define TEST_RTOL_JAKSTAT 1e-06
```

Comment:
does not need to be jakstat specific, with rtol 1e-8, I would not expect more than 1e-6 for any of the models